opensearch-operator/pkg/builders/dashboards.go

```diff
@@ -92,7 +95,8 @@ func NewDashboardsDeploymentForCR(cr *opsterv1.OpenSearchCluster, volumes []core
 						{
 							Name: "dashboards",
 							//	Image: "docker.elastic.co/kibana/kibana:" + cr.Spec.General.Version,
-							Image: "opensearchproject/opensearch-dashboards:1.0.0",
+							Image:           image.GetImage(),
```

Comment:
Based on the discussion with Opensearch devs the Dashboards versions are not going to match up with Opensearch versions.  I've added in a separate configuration to set the Dashboards version, which this will use.  In the future we may want to to some validation against the Opensearch version to make sure they are compatible.

Reply:
maybe we should force that Dashboard version must be equal to OpenSearch version. to avoid error of uncotpatible versions.

Reply:
I don't think we can.  For example in testing I was upgrading to Opensearch version `1.2.3` but this version doesn't exist for Dashboards; the latest is `1.2.0`.  I think maybe in the future we can do some comparison of versions using the semver package.